src/nsrt_learning/strips_learning/oracle_learner.py

```diff
@@ -0,0 +1,28 @@
+"""Oracle for STRIPS learning."""
+
+from typing import List
+
+from predicators.src.envs import get_or_create_env
+from predicators.src.ground_truth_nsrts import get_gt_nsrts
+from predicators.src.nsrt_learning.strips_learning import BaseSTRIPSLearner
+from predicators.src.settings import CFG
+from predicators.src.structs import Datastore, PartialNSRTAndDatastore, \
+    STRIPSOperator
+
+
+class OracleSTRIPSLearner(BaseSTRIPSLearner):
```

Comment:
excellent! i'm glad this is so simple, with your refactoring

Reply:
lol it would have been this simple anyway!